### ISSUE TITLE
RStudio freezes when `verbose` !=0 in `fit`

### ISSUE THREAD

**Issue by BLario** [rstudio/keras#1301]
Very often my whole RStudio becomes all white and freezes after a long Keras model training. This is quite annoying because you need to restart RStudio, loosing all your environment.

After lots of head scratching I figured out this is very probably caused by the convenient `verbose` argument in `fit`: when it's 1 or 2 the Viewer shows the model loss live, and when training is done the process `QtWebEngineProcess` (which I guess is here to render the live view) eats up much of the computer CPU and RAM, which in turn leads to a white RStudio interface.

**Comment by t-kalinowski** [comment]
Hi, thanks for reporting. 
I haven't observed this behavior, so I'll need some additional information to be able to reproduce it.

What OS and platform are you on? How long does model training work before before the freeze? Do you train with any callbacks? Do you know when in the training it freezes? Does the model continue training in the background after RStudio freezes, or model training itself? Do you observe similar freezes outside RStudio?

**Comment by BLario** [comment]
Hi, thanks for your reply:
- I'm on macOS Monterey 12.1
- it happens right after training is done (never during), maybe 10-20 sec after (and RStudio is not usable during this time)
- here are my callbacks:
```
callbacks = list(callback_early_stopping(monitor = "val_loss",
                                             patience = 3,
                                             restore_best_weights = T),
                     callback_reduce_lr_on_plateau(monitor = "val_loss",
                                                   factor = 0.2,
                                                   patience = 3,
                                                   verbose = 0,
                                                   mode = "auto",
                                                   min_delta = 1e-04,
                                                   cooldown = 0,
                                                   min_lr = 0),
                     callback_tensorboard(paste0("./", model_name)))
```
- I don't observe similar freezes outside RStudio

**Comment by t-kalinowski** [comment]
A few more questions to help me track this down:

+ Are you on an Intel Mac or M1 Mac?
+ How long is the model training before the freeze? (both in # of epochs, and time duration)
+ How many metrics is the keras model tracking during training?
+ What RStudio version are you on?
+ What TF and Python version? (reported by `tensorflow::tf_config()`
+ Are you on the dev version or CRAN release version of R packages tensorflow, keras, and reticulate.


Thank you!

**Comment by BLario** [comment]
- Intel Mac
- in my case training takes approx. 1h, for 10-15 epochs
- I use accuracy and `tf$keras$metrics$AUC()`
- R version 4.1.2 
- R Studio version Ghost Orchid
- TF 2.7 Python 3.7
- CRAN release
- 
Thank you!

**Comment by YoshitakaAdachi** [comment]
I also had a similar error.

At the 1st epoch, code seems to run smoooly, but just after 2nd epoch in training, RStudio freezes. 
 Epoch 1/30
254/254 [==============================] - 136s 502ms/step - loss: 0.2094 - dice_coef: 0.8755 - val_loss: 1.6847 - val_dice_coef: 0.0591 - lr: 1.0000e-04
Epoch 2/30
254/254 [==============================] - 103s 405ms/step - loss: 0.0933 - dice_coef: 0.9423

As I followed BLario's comment, I set verbose=0 in callback as follows.
callbacks_list <- list(
  callback_tensorboard("logs_r"),
  callback_early_stopping(monitor = "val_loss",
                          min_delta = 1e-4,
                          patience = 3,
                          verbose = 0,
                          mode = "auto"),
  callback_reduce_lr_on_plateau(monitor = "val_loss",
                                factor = 0.2,
                                patience = 3,
                                verbose = 0,
                                #epsilon = 1e-4,
                                mode = "auto"),
  callback_model_checkpoint(filepath = "weights_r/unet128_{epoch:02d}.h5",
                            monitor = "val_loss",
                            save_best_only = TRUE,
                            save_weights_only = TRUE, 
                            mode = "auto" )
)

I appreciate your kind suggestion. Thank you in advance.
